pkg/controllers/hpa/hpa_controller.go

```diff
@@ -93,9 +93,6 @@ func (c *HorizontalPodAutoscalerController) buildWorks(hpa *autoscalingv1.Horizo
 			Name:       workName,
 			Namespace:  workNamespace,
 			Finalizers: []string{util.ExecutionControllerFinalizer},
-			OwnerReferences: []metav1.OwnerReference{
-				*metav1.NewControllerRef(hpa, hpa.GroupVersionKind()),
-			},
```

Comment:
This will cause a problem that the work generated by hpa will remain.

Can you also add the logic of deleting work synchronously when the HA deletes work?

Reply:
@XiShanYongYe-Chang Thanks for your review

Yes you are right, but I cannot figure out how to delete works because the reconcile request only contains hpa namespace and name, it's impossible to find related works only by them
```go
// Request contains the information necessary to reconcile a Kubernetes object.  This includes the
// information to uniquely identify the object - its Name and Namespace.  It does NOT contain information about
// any specific Event or the object contents itself.
type Request struct {
	// NamespacedName is the name and namespace of the object to reconcile.
	types.NamespacedName
}
```
How about building `ResourceBinding` instead of `Work` when creating a hpa?